src/app/page.tsx

```diff
@@ -1,12 +1,10 @@
-import Link from "next/link";
-
 import { RegionCodeSearch } from "./RegionCodeSearch";
 
 export default async function Home() {
   return (
     <>
       <h1 className="mb-12 text-4xl">
-        Recent bird sightings by region
+        Recent bird sightings by region{" "}
```

Comment:
Thought I'd tidy this up by removing the unused import and adding the space to keep it uniform with the recent/ page!

Reply:
Nice broken window fix 🙏